stories/9-ProgressBar.stories.tsx

```diff
@@ -0,0 +1,206 @@
+import React from "react";
+import {storiesOf} from "@storybook/react";
+
+import ProgressBar from "../src/progress-bar/ProgressBar";
+
+const colors = {
+  background: "#A6A6A6",
+  track: "#2F76FF",
+  completed: "#00B300"
+};
+
+const progressBarStory = (
+  <style>{`
+  .progress-bar-story {
+    height: 51.2px;
+  }
+`}</style>
+);
+
+storiesOf("Progress Bar", module)
```

Comment:
Stories added here are misleading. Storybook example should just demonstrate the use of different available props to the component. So we can have the following examples:

```
ProgressBar with different percentages
ProgressBar with height overridden by CSS
ProgressBar with children
```

There aren't any children on "top", or "bottom". Those aren't children for the ProgressBar component and is irrelevant to the component itself.